Casks/m/mcloud.rb

```diff
@@ -0,0 +1,34 @@
+cask "mcloud" do
+  version "4.1.1,001"
+  sha256 "9221f85930576b22e92e3818b486735da55e2d6c9a4c6de2061a3ad7fc9ddcbf"
```

Comment:
```suggestion
  version "4.1.1,000"
  sha256 "3640839b679f5bc8ecb46ba16dc6bfcc64bbed6241f3e0f55cff64c1960be78a"
```

If we move forward with parsing the `version` parts from the URL in the `livecheck` block, this would be the version and checksum.